wdtk-client/src/test/java/org/wikidata/wdtk/client/JsonSerializationActionTest.java

```diff
@@ -60,6 +60,7 @@ public void testDefaults() {
 		assertTrue(action instanceof JsonSerializationAction);
 		assertFalse(action.needsSites());
 		assertTrue(action.isReady());
+		assertEquals(action.getActionName(), "JsonSerializationAction");
 	}
 
```

Comment:
I don't understand what you mean with "setting a report file". The report file is created by the createReport method in the Client. The report messages consisting of the action name and the getReport result are generated in this method, too. So I don't know to test it. It is also not possible to mock this, or something like this...